dist/index.html

```diff
@@ -3,16 +3,25 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://kit.fontawesome.com/f836ddcdbf.js" crossorigin="anonymous"></script>
 	<title>To Do List</title>
 <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
 <body>
-	<section>
-    <div class="xlist" >
-      <div class="xlist_title x">Today's To Do</div>
-      <div class="xlist_input x"><input type="text" value="" placeholder="Add to your list..." ></div>
-      <div class="xlist_lists " id="list"></div>
-      <div class="xlist_erase x" id="xlist_erase">Clear all completed</div>
-    </div>
-  </section>
+	<section class="section">
+        <div class="container">
+            <div class="heading">
+                <h3 class="list-header">Today's To Do</h3>
+                <span><i class="fa-solid fa-rotate"></i></span>
+            </div>
```

Comment:
Optional - I would highly recommend that you commit more often to be able to track progress and fix bugs 👍🏾